### PR TITLE
chore: Upgrade to 4.0.9 for SDK fixes

### PR DESCRIPTION
**Pull request by alexandreaco**
Includes... 
- fixes decrypting old messages in the transaction history
- adds @radixdlt/crypto to correctly parse unencrypted messages in history

### CODE REVIEW

**Review by SK-Sam — APPROVED** [review]
Hey! I see my older messages 😄 

Screen Shot 2022-01-06 at 10 17 05 AM

**Comment by saminakh** [comment]
> Hey! I see my older messages 😄
> 
> Screen Shot 2022-01-06 at 10 17 05 AM

One of those transactions looks familiar 👀

File: src/composables/useTokenBalances.ts [hunk 1]
```diff
@@ -18,7 +18,6 @@ export default function useTokenBalances (radix: ReturnType<typeof Radix.create>
       // Don't save duplicates
       tokenBalancesRes.account_balances.liquid_balances.map((balance) => {
         if (!relatedTokens.value.find((t) => t.rri.equals(balance.token_identifier.rri))) {
-          console.log('getting info for', balance)
```
Comment (saminakh) [inline src/composables/useTokenBalances.ts:21]: 🧹

File: src/composables/useWallet.ts [hunk 2]
```diff
@@ -85,7 +85,7 @@ const createWallet = (mnemonic: MnemomicT, pass: string, network: Network) => {
     hasWallet.value = true
     setWallet(newWallet)
     saveDerivedAccountsIndex(0, network)
-    persistNodeUrl('https://sandpitnet-gateway.radixdlt.com')
+    persistNodeUrl('https://stokenet-gateway.radixdlt.com')
```
Comment (saminakh) [inline src/composables/useWallet.ts:88]: Would it be easier to define a default node somewhere once and then retrieve it in all these places, so you don't have to keep writing the same url multiple times?

**Review by saminakh — APPROVED** [review]
Code LGTM! And appears to be having as expected locally for me!

File: src/composables/useWallet.ts (hunk 2 above)
Comment (alexandreaco) [reply src/composables/useWallet.ts]: That's a great idea!

**Pull request merged**